mmv1/products/netapp/volume.yaml

```diff
@@ -0,0 +1,405 @@
+              Set the minute of the hour to create the snapshot (0-59), defaults to the top of the hour (0).
+      - !ruby/object:Api::Type::NestedObject
+        name: 'dailySchedule'
+        description: |-
+          Daily schedule policy.
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: 'snapshotsToKeep'
+            description: |-
+              The maximum number of snapshots to keep for the daily schedule.
+            required: true
+          - !ruby/object:Api::Type::Integer
+            name: 'minute'
+            description: |-
+              Set the minute of the hour to create the snapshot (0-59), defaults to the top of the hour (0).
+          - !ruby/object:Api::Type::Integer
+            name: 'hour'
+            description: |-
+              Set the hour to create the snapshot (0-23), defaults to midnight (0).
+      - !ruby/object:Api::Type::NestedObject
+        name: 'weeklySchedule'
+        description: |-
+          Weekly schedule policy.
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: 'snapshotsToKeep'
+            description: |-
+              The maximum number of snapshots to keep for the weekly schedule.
+            required: true
+          - !ruby/object:Api::Type::Integer
+            name: 'minute'
+            description: |-
+              Set the minute of the hour to create the snapshot (0-59), defaults to the top of the hour (0).
+          - !ruby/object:Api::Type::Integer
+            name: 'hour'
+            description: |-
+              Set the hour to create the snapshot (0-23), defaults to midnight (0).
+          - !ruby/object:Api::Type::String
+            name: 'day'
+            description: |-
+              Set the day or days of the week to make a snapshot. Accepts a comma separated days of the week. Defaults to 'Sunday'.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'monthlySchedule'
+        description: |-
+          Monthly schedule policy.
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: 'snapshotsToKeep'
+            description: |-
+              The maximum number of snapshots to keep for the monthly schedule
+            required: true
+          - !ruby/object:Api::Type::Integer
+            name: 'minute'
+            description: |-
+              Set the minute of the hour to create the snapshot (0-59), defaults to the top of the hour (0).
+          - !ruby/object:Api::Type::Integer
+            name: 'hour'
+            description: |-
+              Set the hour to create the snapshot (0-23), defaults to midnight (0).
+          - !ruby/object:Api::Type::String
+            name: 'daysOfMonth'
+            description: |-
+              Set the day or days of the month to make a snapshot (1-31). Accepts a comma separated number of days. Defaults to '1'.
+  - !ruby/object:Api::Type::NestedObject
+    name: restoreParameters
```

Comment:
Can this appear in a test?

Reply:
No. This is a parameter for Volume Backup. The resources for backup (backupPolicies, backupVaults and backupVaults.backups) are new their creation is not scheduled yet.

Reply:
Should we consider excluding this from implementation for now? We can implement it after the resources for backup are added to the provider since we can't test the creation of volume from backup and can not test if the feature is working or not for now.